test/py/cmdlib/instance_unittest.py

```diff
@@ -259,7 +261,7 @@ def testAutoIpNoNameCheck(self):
                          ip_check=False,
                          name_check=False)
     self.ExecOpCodeExpectOpPrereqError(
-      op, "IP address set to auto but name checks have been skipped")
+      op, "IP address set to auto but name checks not enabled")
```

Comment:
"but the name checks are not enabled" (add verb to mirror the first half of the sentence).